data/interfaces/default/settings.html

```diff
@@ -479,7 +479,12 @@
                         </div>
                         <div class="checkbox">
                             <label>
-                                <input type="checkbox" name="movie_notify_enable" id="movie_notify_enable" value="1" ${config['movie_notify_enable']}> Enable Movie and TV Notifications
+                                <input type="checkbox" name="movie_notify_enable" id="movie_notify_enable" value="1" ${config['movie_notify_enable']}> Enable Movie Notifications
+                            </label>
+                        </div>
+                        <div class="checkbox">
+                            <label>
+                                <input type="checkbox" name="tv_notify_enable" id="tv_notify_enable" value="1" ${config['tv_notify_enable']}> Enable TV Show Notifications
```

Comment:
You need to enable this after an update if the user had the old global video notifications option enabled. After updating on my system it disabled TV notifications, people going to wonder why their notifications are broken.